_posts/en/newsletters/2020-12-09-newsletter.md

```diff
@@ -120,12 +120,56 @@ just be more laid back.-->{% endcomment %}
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME:jonatack
+[Periodically make block-relay connections and sync headers][review club
+#19858] is a PR ([#19858][Bitcoin Core #19858]) by Suhas Daftuar to make eclipse
+attacks more difficult by increasing the existing outbound peer rotation beyond
+the current logic of stale tip detection and replacement with a new full-relay
+connection. The proposal initiates outbound connections every 5 minutes on
+average to sync headers and potentially learn of new blocks, in which case it
+rotates out an existing block-relay-only peer in favor of the new peer. This
+would raise the cost to an attacker to sustain a partitioning attack against a
+node, because as long as the node has at least one honest peer in its addrman,
+it should always eventually find the honest chain with the most work.
+
+Most of the discussion focused on understanding the proposed change.
 
 {% include functions/details-list.md
-  q0="FIXME"
-  a0="FIXME"
-  a0link="https://bitcoincore.reviews/19055-2#FIXME"
+  q0="What are block-relay-only connections?"
+  a0="Block-relay-only connections are a type of connection introduced in
+      [Bitcoin Core #15759][Bitcoin Core #15759] that does not participate in
+      transaction or address relay---it only relays blocks."
```

Comment:
similar to above, s/it/they

Reply:
yes, here as written it could go either way, not sure